src/tests/utils/render.test-utils.ts

```diff
@@ -0,0 +1,20 @@
+import { render, type RenderResult } from "@testing-library/svelte";
+import type { ComponentProps, ComponentType, SvelteComponent } from "svelte";
+
+export const renderWithEvents = <C extends SvelteComponent>(
```

Comment:
FYI, in nns-dapp we already have a custom `render` function:
https://github.com/dfinity/nns-dapp/blob/73490d2e89f8b73b33463552b9bfd692d0dd3433/frontend/src/tests/utils/svelte.test-utils.ts#L5

So having a second custom `render` function might not work and it probably makes more sense there to include both functionality in the same `render` function.

With that in mind, I think it might also make sense to just call this function `render` instead of `renderWithEvents` and keep it as a catch-all for things we may have to do to deal with quirks of the underlying `render` function. Perhaps we could even make all calls to `render` go through this (in another PR) so that in the future making adjustments is even easier.

No strong opinion but something to consider.

Reply:
> So having a second custom render function might not work 

I'm not sure what you mean. Both are currently unrelated and exist in different projects. Of course, if we notice during the Svelte v5 migration that such a utility is needed in the NNS dapp codebase, we might consider reusing the same function. However, even then, I'm not sure the NNS dapp should rely on a function from Gix Cmp for testing purposes, kind of feel it's two separate concerns.

> keep it as a catch-all for things 

I feel like the proposed approach introduces fewer changes, which makes the migration easier to achieve. I'm also unsure how the use of events will evolve in the future.

But if you would like to give it a try, not against for sure!

Reply:
Sorry for the long rambling. The only thing I was proposing now is to name the function just `render` instead of `renderWithEvents`. The rest was just trying to explain why I think it makes sense. But if it doesn't make sense to you, it's not important.

Reply:
Ah! Sure!

Reply:
Done